Description: use libdir provided by --libdir
 The old implementation does not support multi-arch
 library locations, like /usr/lib/arm-linux-gnueabi.
Last-Update: 2011-09-06
Author: Sebastian Reichel <sre@debian.org>
Bug-Debian: http://bugs.debian.org/640657
diff --git a/configure.ac b/configure.ac
index 8615f44..d7fbf6d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -41,12 +41,10 @@ PKG_CHECK_MODULES(FSOBASICS,
 AC_SUBST(FSOBASICS_CFLAGS)
 AC_SUBST(FSOBASICS_LIBS)
 
-# gather libdir
-if test "x$prefix" != "xNONE"; then
-  pkglibdir='${prefix}/lib/cornucopia'
-else
-  pkglibdir='/usr/local/lib/cornucopia'
-fi
+test "x$prefix" = xNONE && prefix=$ac_default_prefix
+test "x$libdir" = xNONE && libdir=$prefix/lib
+pkglibdir=$libdir/cornucopia
+
 eval PACKAGE_LIBDIR="${pkglibdir}"
 AC_DEFINE_UNQUOTED([PACKAGE_LIBDIR], ["${PACKAGE_LIBDIR}"], [Module directory])
 
